cvat/apps/lambda_manager/views.py

```diff
@@ -98,6 +102,8 @@ def __init__(self, gateway, data):
                 "`{}` lambda function has non-unique labels".format(self.id),
                 code=status.HTTP_404_NOT_FOUND)
         self.labels = labels
+        # mapping of labels and corresponding supported attributes
+        self.attributes = {item['name']: item.get('attributes', []) for item in spec}
```

Comment:
@mikhail-treskin , we need to keep attributes per label. For example, `person` label with `age`, `gender` attributes and `car` label with `model` and `type` attributes. We should have a separate list for each label. Not just `['age', 'gender', 'model', type'].